packages/media/tests/webrtc/P2pRtcManager.spec.ts

```diff
@@ -591,25 +591,25 @@ describe("P2pRtcManager", () => {
 
     describe("accept", () => {
```

Comment:
acceptNewStream?

Reply:
Fixed in [media: rename test contextual function](https://github.com/whereby/sdk/pull/378/commits/367ba5b8b0d4fb6fa62129082a0fa8445491e03b)